packages/create-sitecore-jss/src/templates/nextjs-personalize/src/lib/extract-path/plugins/personalize.ts

```diff
@@ -0,0 +1,14 @@
+import { normalizePersonalizedRewrite } from '@sitecore-jss/sitecore-jss-nextjs';
+import { Plugin } from '..';
+
+class PersonalizePlugin implements Plugin {
+  /**
+   * Extract normalized Sitecore item path from query
+   * @param {string} path
+   */
```

Comment:
Can probably remove the jsdoc comment here (we don't do this for other plugin's execs, "query" no longer makes sense anyway)
```suggestion
```